src/pages/ViewListing.jsx

```diff
@@ -233,9 +234,13 @@ const ViewListing = () => {
   }, [contracts.MarketContract, contracts.ZoombiesContract, contracts.WMOVRContract, contracts.ZoomContract, auctionId]);
 
   useEffect(() => {
-    if (auctionId) {
-      getOffers(auctionId)
-    }
+    const token = PubSub.subscribe("BID_EVENT", (msg, data) => {
+      if (auctionId) {
+        getOffers(auctionId)
```

Comment:
If we are refetching from the server when a new event happens, can we guarantee that the new event has been stored in the DB?

Reply:
Probably not